src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -168,4 +229,25 @@ private static boolean isSwitchExpression(DetailAST ast) {
                 || ast.getParent().getParent().getType() == TokenTypes.EXPR;
     }
 
+    /**
+     * Iterates through siblings of an ast node, evaluating given predicate for each
+     * sibling and returns the first matching node. This operation includes
+     * evaluation of the given node.
+     *
+     * @param ast the ast to check self and siblings of
+     * @param predicate the expression to evaluate
+     * @return first matching node, or null if matching node is not present
+     */
+    private static DetailAST findAstByPredicate(DetailAST ast, Predicate<DetailAST> predicate) {
```

Comment:
It made sense to extract the logic of finding a particular ast by some predicate, almost all private methods in this check have similar logic now that we can have syntax like `case null, default:`. We just check all case label elements for each case.

Reply:
Why not `TokenUtil#findFirstTokenByPredicate` ?

Reply:
> Why not `TokenUtil#findFirstTokenByPredicate` ?

Because I completely forgot about it :) Thank you.

Reply:
Done.